tests/test_nanoevents_vector.py

```diff
@@ -519,22 +534,36 @@ def test_inherited_method_transpose(lcoord, threecoord, twocoord):
         )
     elif twocoord == "PolarTwoVector":
         c = ak.zip(
-            {"r": [-10.0, 13.0, 15.0], "phi": [1.22, -1.0, 1.0]},
+            {"rho": [-10.0, 13.0, 15.0], "phi": [1.22, -1.0, 1.0]},
```

Comment:
This renaming of `r` to `rho` is something we'll need to remember to include in the release notes.
Perhaps a migration guide is in order? Are there any other user-facing changes we'll have to advertise?

Reply:
This the only one that isn't resolved by reading vector's documentation.

I think this one will be a fairly low hit-rate, since most people would specify a polar two vector using pT and phi, which is the same in both cases.